{"name": "Successful Login", "status": "broken", "statusDetails": {"message": "playwright._impl._errors.Error: It looks like you are using Playwright Sync API inside the asyncio loop.\nPlease use the Async API instead.\n", "trace": "  File \"C:\\Users\\a90011969\\Playwright-Python\\Playwright-Python\\playwright-behave-env\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\a90011969\\Playwright-Python\\Playwright-Python\\playwright-behave-env\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"steps\\user_login.py\", line 7, in login\n    context.playwright = sync_playwright().start()\n                         ~~~~~~~~~~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\a90011969\\Playwright-Python\\Playwright-Python\\playwright-behave-env\\Lib\\site-packages\\playwright\\sync_api\\_context_manager.py\", line 84, in start\n    return self.__enter__()\n           ~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\a90011969\\Playwright-Python\\Playwright-Python\\playwright-behave-env\\Lib\\site-packages\\playwright\\sync_api\\_context_manager.py\", line 47, in __enter__\n                raise Error(\n    ...<2 lines>...\n                )\n"}, "steps": [{"name": "Given I navigate to the login page", "status": "broken", "statusDetails": {"message": "playwright._impl._errors.Error: It looks like you are using Playwright Sync API inside the asyncio loop.\nPlease use the Async API instead.\n", "trace": "  File \"C:\\Users\\a90011969\\Playwright-Python\\Playwright-Python\\playwright-behave-env\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\a90011969\\Playwright-Python\\Playwright-Python\\playwright-behave-env\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"steps\\user_login.py\", line 7, in login\n    context.playwright = sync_playwright().start()\n                         ~~~~~~~~~~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\a90011969\\Playwright-Python\\Playwright-Python\\playwright-behave-env\\Lib\\site-packages\\playwright\\sync_api\\_context_manager.py\", line 84, in start\n    return self.__enter__()\n           ~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\a90011969\\Playwright-Python\\Playwright-Python\\playwright-behave-env\\Lib\\site-packages\\playwright\\sync_api\\_context_manager.py\", line 47, in __enter__\n                raise Error(\n    ...<2 lines>...\n                )\n"}, "start": 1739335095636, "stop": 1739335095638}, {"name": "When I enter valid credentials", "status": "skipped", "start": 1739335095640, "stop": 1739335095640}, {"name": "And I submit the form", "status": "skipped", "start": 1739335095640, "stop": 1739335095640}, {"name": "Then I should see a success message", "status": "skipped", "start": 1739335095640, "stop": 1739335095640}], "start": 1739335095635, "stop": 1739335095640, "uuid": "3367025b-81fd-4029-82b2-3c254a2a54d1", "historyId": "58530cb6d300b9be77ad6f1da67f27a5", "fullName": "User Login: Successful Login", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "User Login"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}